### DISCUSSION TITLE
Where we can give feedback regarding teams?

### DISCUSSION THREAD

**Discussion by beingminimal** [community/community#53291] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

HI There!

We haven't found any category to give feedback regarding the team in github community.
If we don't have any category then please add a new category for teams.

Thanks

**Comment by beingminimal** [comment]
@queenofcorgis can you please help with this? It's a request. Thanks

**Reply by queenofcorgis** [reply]
Hi there - would you mind elaborating your request please so I can better understand it?

**Reply by beingminimal** [reply]
in case anyone wants to give feedback regarding github team feature which users are currently using in github organisation.
Here is the doc link https://docs.github.com/en/organizations/organizing-members-into-teams/about-teams
but i couldn't find discussion categories for either organisation or teams to give feature requests.

**Comment by beingminimal** [comment]
@queenofcorgis Please create discussion categories for teams & organisations in github community.
It's humble request so millions of github users can place feature request there. Thank you.

**Comment by beingminimal** [comment]
@labudis Can you please help here. Thanks

**Reply by labudis** [reply]
@beingminimal there doesn't seem to be a specific category related to Teams, but you could post it in General and choose "Product feedback" in the drop-down. Thanks!

**Reply by beingminimal** [reply]
@labudis We must add a separate category for teams because the functionality of Github teams is so important. What are your thoughts?
